test/spicepods/tpch/sf1/accelerated/indexes/file[parquet]-duckdb[file]-indexes.yaml

```diff
@@ -0,0 +1,82 @@
+version: v1
+kind: Spicepod
+name: file[parquet]-duckdb[file]-indexes
+datasets:
+  - from: file:data/customer.parquet
+    name: customer
+    acceleration:
```

Comment:
We should add primary_key to each of these, and also have at least one compound/clustered index.

Reply:
Same with the others. Also, I think it's really only worth running these at SF-10 or SF-100.